src/fu-common-guid.c

```diff
@@ -131,3 +131,29 @@ fu_common_guid_from_string (const gchar *str)
 	return fu_common_guid_from_data ("6ba7b810-9dad-11d1-80b4-00c04fd430c8",
 					(const guint8 *) str, strlen (str), NULL);
 }
+
+/**
+ * fu_common_guid_get_plausible:
+ * @buf: a buffer of data
+ * @blksz: the start address to get the GUID from
+ *
+ * Checks whether a chunk of memory looks like it can be an UEFI guid.
+ *
+ * Returns: TRUE if it looks like a GUID, FALSE if not
+ *
+ * Since: 1.2.5
+ **/
+gboolean
+fu_common_guid_get_plausible (const guint8 *buf, guint16 addr_start)
```

Comment:
I think it wants to be `_is_plausible()` as it's not actually returning the GUID.